### DISCUSSION TITLE
Quality of videos with small sized text

### DISCUSSION THREAD

**Discussion by theDestI** [remotion-dev/remotion#1100] [General]
Hi, I'm trying to render a video with a CSS code in it and here's how it looks on the preview:
![image](https://user-images.githubusercontent.com/17301439/178113365-4d3f3b24-ba18-453d-86f1-05404f12e9f3.png)
While the render version looks like this:
![image](https://user-images.githubusercontent.com/17301439/178113409-0b68edb5-574e-479c-a832-dd115e1699c7.png)

Here's what my remotion config looks like:

`Config.Rendering.setImageFormat('jpeg');
Config.Output.setOverwriteOutput(true);

Config.Output.setCrf(1);
Config.Rendering.setQuality(100);
Config.Rendering.setConcurrency(1);`

I've tried to experiment with settings a lot but I was unable to achieve something substantially better. Are there any good examples of how to render small-sized text or is it the expected quality due to the fact that there are losses during the screenshot making?

**Comment by JonnyBurger** [comment]
Hi @theDestI!

I assume it happens because you preview on a 2x pixel density display (e.g Retina display on a Macbook), but the resolution of the video is not high enough to render the text sharply.

You can scale the render to output a more sharp text! https://www.remotion.dev/docs/scaling#how-to-scale

**Comment by theDestI** [comment]
Thanks a lot! 

It's a bit unobvious, though. Do you think it would be a good addition to docs to highlight a case like this?
